concepts/50 React Components/02 Create a DevExtreme Application/30 Configure Navigation Menu Items.md

```diff
@@ -0,0 +1,21 @@
+Edit the `src\app-navigation.js` file to configure navigation menu items. Each item configuration can have the following fields:
```

Comment:
I think we should clarify that it's incorrect to specify both `path` or `items` field at the same time.
A navigation menu item can be a folder or a page. If it is a folder - specify `items`, if it s a page - specify `path`.